resolwe_bio/tests/processes/test_import.py

```diff
@@ -37,14 +37,14 @@ def external_test_sra(self):
             import_sra,
             "fastqc_url",
             [
-                {
-                    "file": "fastqc/SRR1661332_fastqc/fastqc_report.html",
-                    "refs": ["fastqc/SRR1661332_fastqc"],
-                },
                 {
```

Comment:
Elements in `fastqc_url` should be listed in the order that corresponds to the order of SRR references in the input. If this order is not platform invariant, the relevant process needs to be corrected.